scripts/run_training.py

```diff
@@ -52,6 +55,8 @@ def setup_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(description="Train a delphi model")
     parser.add_argument(
         "--config_files",
+        "--config_file",
+        "-c",
```

Comment:
```suggestion
        "--config",
        "-c",
```
I don't care about backward compatibility, no one is using this yet